### ISSUE TITLE
unable to log on 

### ISSUE THREAD

**Issue by Len956** [WoWManiaUK/Redemption#4508]
**Links:**

**What is Happening:**

Hello
I'm rejoining today , however I can't seem to log into the actual game . When I try I get this message & I also rejoined the VIP club ( Forbidden (#403)
You are not allowed to perform this action.)
I looked through the forum but I can't find any info .
Please help

Len
**What Should happen:**

**Comment by Rushor** [comment]
download the latest client from website :)

**Comment by Len956** [comment]
Hello Rushor

I tried to download & I get this error message (Forbidden (#403)
You are not allowed to perform this action. )  I get this when I click on the Client drop down to show Connect .
I then try to open a account & I get that message . Is it something very obvious that I don't see or is it something else? I had previously played on your server & everything worked fine , I even took out the VIP subscription before I started the game but now I can't seem to get back playing.

Thank You

Len

**Comment by Len956** [comment]
Hello Rushor

I tried to download & I get this error message (Forbidden ( <https://github.com/WoWManiaUK/Redemption/issues/403> #403)
You are not allowed to perform this action. ) I get this when I click on the Client drop down to show Connect .
I then try to open a account & I get that message . Is it something very obvious that I don't see or is it something else? I had previously played on your server & everything worked fine , I even took out the VIP subscription before I started the game but now I can't seem to get back playing.

Thank You

Len
